api/api.yaml

```diff
@@ -112,6 +94,31 @@ paths:
         '500':
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
 
+  /control/flows:
+    post:
+      tags: ['Control']
```

Comment:
shouldn't it belong to Config tag instead of Control ? since we're essentially modifying flow properties.

Reply:
By looking at the existing tags (esp. with reference to /control/link, /control/routes), my understanding is tag is more associated with classification of type of api and not based on its operands - is it not ?
From that perspective , I have kept it as Control which seems in line.